FAQ.md

```diff
@@ -0,0 +1,52 @@
+# FAQ - Frequently Asked Questions
+
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+
+**Table of Contents**
+
+- [FAQ - Frequently Asked Questions](#faq---frequently-asked-questions)
+  - [Installation](#installation)
+  - [Trading](#trading)
+  - [Wallet](#wallet)
+  - [Contact us](#contact-us)
+
+<!-- markdown-toc end -->
+
+## Installation
+
+Q: Do I need to use Umbrel to use Itchysats?
```

Comment:
🙃 

A little nitpicky, but I see "Itchysats", "ItchySats" and "itchysats" being used in this document.

Reply:
looking at other public stuff, like Twitter etc, I will unify it to "ItchySats".

Reply:
We agreed on `ItchySats` at some point, but it might not be consistent in all our communication.

Reply:
I will make sure I use that too, because I have not been consistent.